### PR TITLE
Fixed missing str vars from browser page

### LINKED ISSUE #2207: STRs variants don't show up in current master
Even if they are loaded and in database

### PR DESCRIPTION
**Pull request by northwestwitch**
Bug fix #2207 

**How to test**:
1. On a local instance of scout, load the demo case

**Expected outcome**:
1. Strs should show in their respective page

**Review:**
- [x] code approved by
- [x] tests executed by

### CODE REVIEW

File: scout/server/blueprints/variants/views.py [hunk 1]
```diff
@@ -208,8 +208,10 @@ def str_variants(institute_id, case_name):
             ("position", pymongo.ASCENDING),
         ]
     )
-
-    data = controllers.str_variants(store, institute_obj, case_obj, variants_query, page)
+    result_size = store.count_variants(case_obj["_id"], {}, None, category)
```
Comment (northwestwitch) [inline scout/server/blueprints/variants/views.py:211]: To make this compliant with the str filters, just substitute {} with the request.form.data, I guess

File: scout/server/blueprints/variants/controllers.py [hunk 2]
```diff
@@ -203,15 +203,16 @@ def get_manual_assessments(variant_obj):
     return assessments
 
 
-def str_variants(store, institute_obj, case_obj, variants_query, page=1, per_page=50):
+def str_variants(
+    store, institute_obj, case_obj, variants_query, variant_count, page=1, per_page=50
```
Comment (dnil) [inline scout/server/blueprints/variants/controllers.py:207]: :+1: Yep, exactly the same as #2166.

File: scout/server/blueprints/variants/views.py (hunk 1 above)
Comment (dnil) [inline scout/server/blueprints/variants/views.py:211]: Yeah, this should basically the only collision, the rest will be identical, so no worries.

**Review by dnil — APPROVED** [review]

**Comment by dnil** [comment]
I'll just merge this! Objections? 5.. 4..

File: scout/server/blueprints/variants/controllers.py (hunk 2 above)
Comment (northwestwitch) [reply scout/server/blueprints/variants/controllers.py:207]: I see. The "pagination" in the title was a bit of an understatement 😆

**Pull request merged**